client/extensions/wp-super-cache/contents-tab.js

```diff
@@ -180,19 +188,19 @@ class ContentsTab extends Component {
 
 				<div>
 				{ ! isEmpty( get( wpcache, 'cached_list' ) ) &&
-					<CacheStats header="Fresh WP-Cached Files" files={ wpcache.cached_list } />
+					this.renderCacheStats( wpcache.cached_list, translate( 'Fresh WP-Cached Files' ), true, false )
```

Comment:
Hmm, generally I find separate components preferable over `this.render...()` methods. (We had some [components](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/theme/main.jsx) in the Theme Showcase that got pretty badly bloated with loads of `render...()`s, so I might be biased). Also, it's easier to keep track of named props as opposed to a list of 4 unnamed function args (tho the latter could be mitigated by passing a single object arg with named attrs).

Is the motivation here to avoid repetitively specifying props that are always the same, such as `siteId`? That particular case could be resolved by `connect`ing `CacheStats` to `getSelectedSiteId()`.